StarkInfra/StarkInfra/IssuingEmbossingKit/IssuingEmbossingKit.cs

```diff
@@ -0,0 +1,229 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using StarkInfra.Utils;
+
+
+namespace StarkInfra
+{
+    /// <summary>
+    /// IssuingEmbossingKit object
+    /// <br/>
+    /// The IssuingEmbossingKit object displays information on the card and card package designs available to your Workspace.
```

Comment:
The IssuingEmbossingKit object displays information on the embossing kits available to your Workspace.